js/components/utils/hotkeys.vue

```diff
@@ -3,8 +3,10 @@
     @keydown.space="togglePlayback"
     @keydown.j = "playNext"
     @keydown.k = "playPrev"
-    @keydown.f = "search"
-    @keydown.l = "toggleLike"
+    @keydown.h = "playRewind"
+    @keydown.l = "playForward"
```

Comment:
Not sure it's OK to remove the functionality of `toggleLike` completely here. Maybe use another key for the feature?